modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -93,6 +93,117 @@
           end
         end
       end
+
+      describe 'Getting an unauthorized reponse' do
+        response '401', 'Unauthorized' do
+          let(:scopes) { %w[system/claim.read system/claim.write] }
+
```

Comment:
scopes can be moved to the top describe, so that we don't have to declare it for each endpoint test